refinery/templates/registration/registration_form.html

```diff
@@ -93,6 +93,15 @@ <h4>Terms of Service</h4>
 			</label>
 			{% endcomment %}
 
+      {% if form.spam_filter.errors %}
```

Comment:
Wondering if showing errors for hidden fields is necessary if this is a scenario that only spam bots will run into

Reply:
@scottx611x There are form filling extensions which may fill this field. It will annoy the user, but most of our users are manually filling these forms.